test/common/addresses.spec.ts

```diff
@@ -0,0 +1,14 @@
+import { expect } from '../setup'
+
+/* Imports: Internal */
+import { getRandomAddress } from '../../src'
+
+describe('getRandomAddress', () => {
+  beforeEach(async () => {
```

Comment:
Do we know that `Math.random` isn't poisoned for the rest of the tests? This could cause a really annoying bug if another test in the future relies on `Math.random`. This should modify `Math.random` in a `before` and then fix it in an `after`

Reply:
Good point. Will make a PR fixing this.

Reply:
I had this exact concern! In some brief research it seemed like Mocha should box each individual test so there is no test pollution via global variable overrides, although I've seen weird behavior with unresolved Promises leaking in between tests on other projects. That may be a frontend-specific issue? I'll put together a quick test after work to verify Math.random does not just print 8 in other test cases after this test is ran.